data/campaigns/Liberty/scenarios/08_Glory.cfg

```diff
@@ -500,7 +500,7 @@
                 [/message]
                 [message]
                     speaker=Baldras
-                    message= _ "When I was a boy, my father — your grandfather — brought your dad and me to live in Aldril after orcs massacred many in our village. We grew up around here, and we learned many things."
+                    message= _ "When I was a boy, my father — your grandfather — brought your mom and me to live in Aldril after orcs massacred many in our village. We grew up around here, and we learned many things."
```

Comment:
The change makes sense, but I wonder whether the references to Harper's father in S1 and S8 make as much sense now. Considering Harper's parents are only alluded to, it seems a bit weird that Baldras only ever talks about promising Harper's father to look after him. I guess Harper's mother had already died earlier and Harper's father was still around to look after him at that point.

Still, the change here is correct, it's just a minor detail that could be more clear. The original intent might have been for Harper's father to be Baldras' brother (since that way the references to Harper's parents work better), but we probably shouldn't change back to that either at this point, for translators' sanity if nothing else.

Reply:
Another interpretation is, Harper's mother is still alive and Baldras succeeded his brother-in-law as patriarch.

Reply:
I'm not against the change, but I'd prefer Baldras to say 'mother' rather than 'mom'. Questions of formality aside, 'mom' sounds out of place to me even though I'm aware en-US is the default locale - if the story of Wesnoth is supposed to have a medieval tone/setting, then I take any dialogue spoken to favour an older European style over American. It's the same reasoning  I understand for en-GB to favour the older -ize suffixes over the more contemporary -ise influence from French, even though I personally prefer the latter.

If this is all too much nit-picking for you and you choose to reject this suggestion, then that's fine, it's your choice.